core/network/network.go

```diff
@@ -61,10 +61,13 @@ const (
 	// CannotConnect means recently attempted connecting but failed to connect.
 	// (should signal "made effort, failed")
 	CannotConnect
```

Comment:
Maybe we can just get rid of these? I don't think _anyone_ is referencing them...

Reply:
That would also simplify checks to `!= disconnected` (transient ok) `== connected` (transient not ok).

Reply:
It'd be nice to fix these too. Can you remove them?

Reply:
Removing these will be a breaking change (it'll break actual users. IMO, we should deprecate first.

Reply:
SGTM.

Reply:
`CanConnect` and `CannotConnect` deprecated